src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int sideA;
+    private int sideB;
+
+    RightTriangle(String color, int sideA, int sideB) {
+        super(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    @Override
+    public void returnInfo() {
```

Comment:
The `returnInfo` method should be renamed to `draw` to meet the task requirements and to maintain consistency with the concept of separating different behaviors into appropriate interfaces and methods.